src/WorkItems/TestWorkItemData.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.WorkItems
+{
+    public class TestWorkItemData
```

Comment:
>TestWorkItemData [](start = 17, length = 16)

Can't seem to find where this class is being used.

Reply:
Oops, deleted.

---
In reply to: [381458484](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381458484) [](ancestors = 381458484)